README.md

```diff
@@ -1,6 +1,6 @@
 # Package Versions
 
-This utility provides very quick and easy access to installed composer dependency versions.
+This utility provides quick and easy access to version information of composer dependencies as derived from the ```composer.lock``` file which is (re)generated during ```composer install``` or ```composer update```.
```

Comment:
Can you add this information as a second bit, rather than as part of the first sentence? This seems to kill the description of the package, making it a bit more unclear :D